src/containers/Application/Part3.js

```diff
@@ -1,6 +1,169 @@
-import React from 'react'
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Dropdown, Select, TextInput } from '../../components/Input'
+import { H1, QuestionHeading } from '../../components/Typography'
+
+const StyledDropdown = styled(Dropdown)`
+  .react-select__control {
+    margin: 0;
+  }
+`
+
+const QuestionContainer = styled.div`
+  margin-bottom: 6em;
+`
+
+const StyledTextInput = styled(TextInput)`
+  margin: 0.5em 0 0;
+`
 
 // form part 3
-export default ({ children }) => {
-  return <div>{children}</div>
+export default () => {
+  // eslint-disable-next-line no-unused-vars
+  const [selectedOption, setSelectedOption] = useState('')
+  const [otherOption, setOtherOption] = useState('')
+  const [showOther, setShowOther] = useState('')
+  const [selectStates, setSelectStates] = useState({
+    multiselect: {
+      option1: false, // LHD / Hack Camp
+      option2: false, // nwHacks
+      option3: false, // cmd-f
+      option4: false, // cmd-f Phases
+      option5: false, // nwPlus Workshop Series
+      option6: false, // nwPlus Boothing
+    },
+  })
+
+  const options = [
+    { value: '1', label: 'MLH' },
+    { value: '2', label: 'Social media' },
+    { value: '3', label: 'Website' },
+    { value: '4', label: 'Word of mouth' },
+    { value: '5', label: 'Club newsletter' },
+    { value: '6', label: 'Faculty newsletter' },
+    { value: '7', label: 'Professor/in class' },
+    { value: '8', label: 'Other' },
+  ]
+
+  const handleSelect = inputValue => {
+    setSelectedOption(inputValue.label)
+
+    // Show additional input for 'Other'
+    if (inputValue.label === 'Other') {
+      setShowOther(true)
+    } else {
+      setShowOther(false)
+    }
+  }
+
+  return (
+    <>
+      <QuestionContainer>
+        <QuestionHeading>Question 14</QuestionHeading>
+        <H1 size="1.5em">How did you hear about nwHacks?</H1>
+        <StyledDropdown
+          options={options}
+          placeholder={'Select an option'}
+          isSearchable={false}
+          onChange={inputValue => handleSelect(inputValue)}
+          isValid
+        />
+        {showOther && (
+          <div>
+            <div>Please specify</div>
+            <StyledTextInput value={otherOption} onChange={e => setOtherOption(e.target.value)} />
```

Comment:
Can you use the same property to keep track of the `other` value?
```suggestion
            <StyledTextInput value={otherOption} onChange={e => setSelectedStates({...selectedStates, engagementSource: e.target.value)} />
```

Reply:
Also quicc comment! I created a 'Please specify' no outline `TextInput` component - might want to use this after I get my PR merged :) https://github.com/nwplus/livesite/pull/167

Reply:
Thank you! This is great 🔥

Added here! dffb17f